pkg/controllers/netpol/network_policy_controller.go

```diff
@@ -928,7 +941,7 @@ func cleanupStaleRules(activePolicyChains, activePodFwChains, activePolicyIPSets
 		for podFwChain := range activePodFwChains {
 			podFwChainRules, err := iptablesCmdHandler.List("filter", podFwChain)
 			if err != nil {
-
+				glog.Errorf("Failed to list iptable filters")
```

Comment:
```go
glog.Errorf("Failed to list iptable filters: %s", err)
```

Reply:
should this have `metrics.ControllerErrors.WithLabelValues(networkPolicyControllerLabel).Inc()`?